dkg-gadget/src/metrics.rs

```diff
@@ -18,9 +18,20 @@ use prometheus::{register, Counter, Gauge, PrometheusError, Registry, U64};
 /// DKG metrics exposed through Prometheus
 #[derive(Clone)]
 pub(crate) struct Metrics {
-	pub propagated_messages: Counter<U64>,
+	/// Count to total propogated messages
+	pub dkg_propagated_messages: Counter<U64>,
 	/// Current active validator set id
 	pub dkg_validator_set_id: Gauge<U64>,
+	/// Total messages received
+	pub dkg_inbound_messages: Counter<U64>,
+	/// Total error messages received
+	pub dkg_error_counter: Counter<U64>,
+	/// Current session progress witnessed by dkg worker
+	pub dkg_session_progress: Gauge<U64>,
+	/// Keygen retry counter for dkg worker
+	pub dkg_keygen_retry_counter: Counter<U64>,
+	/// The latest block height witnessed by the dkg worker
+	pub dkg_latest_block_height: Gauge<U64>,
```

Comment:
Should we add a metrics for key rotation or is that captured in session progress / keygen retry?